### ISSUE TITLE
Titleize the Address? (TBC)

### ISSUE THREAD

**Issue by beseven** [nhsuk/gp-data-etl#20]
In a similar way to the practice name, do we need to Titleize the address too?
e.g
https://beta.nhs.uk/gp-surgeries/45267
https://beta.nhs.uk/book-a-gp-appointment/results?search=THORKHILL

cc / @mcgrathjoe

**Comment by mcgrathjoe** [comment]
@beseven yep, there's already a card for this in Todo - https://trello.com/c/5RwOdmx1/229-improve-formatting-of-gp-contact-details
